static/src/javascripts/projects/common/modules/onward/us-survey-banner.js

```diff
@@ -0,0 +1,78 @@
+define([
+    'common/views/svgs',
+    'common/utils/config',
+    'common/utils/detect',
+    'common/utils/template',
+    'common/modules/ui/message',
+    'common/modules/user-prefs',
+    'text!common/views/experiments/us-survey-banner.html'
+],function(
+    svgs,
+    config,
+    detect,
+    template,
+    Message,
+    userPrefs,
+    messageTemplate
+){
+
+    var messageId = 'mobile-labs-alert',
+        messageOptions = {
+            siteMessageLinkName: 'mobile labs | message | presidential primary alerts',
```

Comment:
It may be easier to get the interactions out of Ophan if you add a data-component

You could use the option `siteMessageComponentName` to add your preferred component name to the outermost div of the site message